doc/doxygen/scripts/make_gallery.pl

```diff
@@ -163,7 +163,7 @@
         print "<a name=\"ann-$file\"></a>\n";
         print "<h1>Annotated version of $file</h1>\n";
 
-        system $^X, "$cmake_source_dir/doc/doxygen/scripts/program2doxygen", "$gallery_dir/$file";
+        system $^X, "$cmake_source_dir/doc/doxygen/scripts/program2doxygen.pl", "$gallery_dir/$file" , "--prefix=$file";
```

Comment:
You use the name of the file as a prefix, but does that always lead to allowed characters in an HTML anchor?

Reply:
In line 163 it is used as well so I supposed that it would be fine.